app/models/lighthouse/hcc/invoice.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module Lighthouse
+  module HCC
+    class Invoice
+      include Vets::Model
+      attribute :external_id, String
+      attribute :facility, String
+      attribute :latest_billing_ref, String
+      attribute :current_balance, Float
+      attribute :previous_balance, String
+      attribute :previous_unpaid_balance, String
+      attribute :last_updated_at, String
+      attribute :last_credit_debit, Float
+      attribute :url, String
+
+      def initialize(params)
+        @params = params
+        assign_attributes
+      end
+
+      def assign_attributes
+        @facility = @params.dig('resource', 'issuer', 'display')
+        @latest_billing_ref = @params['resource']['lineItem'].first['chargeItemReference']['reference'].split('/').last
```

Comment:
@params.dig is missing from lines 24 and 26. This would prevent `NoMethodError` errors. Should dig be added to those lines?